lib/membership_lists/wikidata.rb

```diff
@@ -0,0 +1,47 @@
+require 'rest-client'
+
+module MembershipLists
+  WIKIDATA_SPARQL_URL = 'https://query.wikidata.org/sparql'.freeze
+
+  class Wikidata
+    def initialize(wikidata_membership_item:)
+      @wikidata_membership_item = wikidata_membership_item
+    end
+
+    def to_a
+      @to_a ||= JSON.parse(
+        sparql_results, symbolize_names: true
+      )[:results][:bindings].map(&method(:sparql_result_to_hash))
+    end
+
+    private
+
+    attr_reader :wikidata_membership_item
+
+    def sparql_query
+      @sparql_query ||= """
+        SELECT ?item ?itemLabel
+          WHERE {
+            ?item wdt:P39 wd:#{wikidata_membership_item}.
+            SERVICE wikibase:label { bd:serviceParam wikibase:language \"[AUTO_LANGUAGE],en\". }
+          }
+"""
+    end
+
+    def sparql_results
+      @sparql_results ||= RestClient.get WIKIDATA_SPARQL_URL, params: { query: sparql_query, format: 'json' }
+    rescue RestClient::Exception => e
+      raise "Wikidata query #{query.inspect} failed: #{e.message}"
+    end
+
+    def sparql_result_to_hash(sparql_result)
```

Comment:
It was quite difficult to puzzle out what was going on here (and I'm still not totally sure I fully understand it, as I don't think I've ever seen the `.map(&method(:foo)` idiom before). 

I think some of my confusion is down to having `sparql_results` and `sparql_result` be conceptually different things (rather than `sparql_results` being simply a list of `sparql_result`s, as I would have expected), and the connection between them being quite indirect. If I'm following along correctly this is really just a helper function included here to avoid us needing a separate class?

I think this would be a bit clearer if `sparql_results` was actually only the results, and the method currently called that was something like `sparql_response`. Whether that's better as the "raw" results straight out of `JSON.parse(sparql_response, symbolize_names: true)[:results][:bindings]`, or processed through this call, I'm not sure, but I'd possibly try having a different method for each, just to make it easier to follow.